Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,249 @@
+# Motor System
+
+## Subsystem Function
+
+This subsystem's role in our design is to move new devices to the checkout door by receiving power from our power subsystem and rotating the platforms holding the devices. This will only be done when the machine senses that there is a device missing from the checkout drawer. After discussing with the mechanical engineering team, we have decided to make the platforms out of acrylic and the hollow tube inside our machine to be either **aluminum** or **pvc**.  
+ 
+## Constraints
```

Comment:
Didn't consider the precision of the positioning. How precise does it need to be and can the motor/gear/belt system achieve it?